docs/UserGuide.md

```diff
@@ -453,46 +449,75 @@ Examples:
 
 ### Adding a task: `add` `-p`
 
-Adds a task or recurring task to a patient.
+You can add a task or recurring task to a patient with the `add` command.
 
-Format: `add -p PATIENT_INDEX d/TASK_DESCRIPTION | <DATE TIME> | <INTERVAL TIME_PERIOD>`
+Format: **`add`** `-p PATIENT_INDEX d/TASK_DESCRIPTION | <DATE TIME> | <INTERVAL TIME_PERIOD>`
 
-* Adds a task to a patient at the specified `PATIENT_INDEX`.
-* `DATE TIME` and `INTERVAL TIME_PERIOD` must follow the criteria defined in [Task parameters](#task-parameters).
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Notes:**
+* `DATE TIME` and `INTERVAL TIME_PERIOD` must follow the criteria defined in [Task parameters](#parameter-constraints).
+* Note that tasks are automatically sorted in chronological order upon being added, i.e if a task on `25-10-22` is added, this task's `TASK_INDEX` would be based on its chronological order in the patients task list. E.g the patient already contains tasks on `24-10-22` and `27-10-22`, the new task will be the 2nd task for the patient after the one on `24-10-22`.
+* If a `TIME` of `2400` if provided, it would simply be translated into `0000` hours of the next day. <br>
```

Comment:
Phrasing seems a bit awkward, and there's a typo. I think this should be put as a warning in a separate box from the info box instead (see formatting in the `Editing data file section`). I think this explanation is enough to omit an example.

```suggestion
<div markdown="block" class="alert alert-warning">

:exclamation: **Caution:**
If you enter a `TIME` of `2400`, then the date and time will be set to `0000` hours of the next day.

</div>
```